Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'awesome_print'
+gem 'devise', '~> 3.4.1'
```

Comment:
Needed to upgrade because I ran into some errors with v3.2 -
```
/Users/josh/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/devise-3.2.4/lib/devise/rails/routes.rb:455:in `ensure in with_devise_exclusive_scope': undefined method `merge!' for #<ActionDispatch::Routing::Mapper::Scope:0x007fa5903a3b20> (NoMethodError)
```